src/components/BorrowerProfile/LendCta.vue

```diff
@@ -558,7 +516,7 @@ export default {
 			this.matchingTextVisibility = this.status === 'fundraising' && this.matchingText && !this.isMatchAtRisk;
 
 			if (this.status === 'fundraising' && this.numLenders > 0) {
-				this.lenderCountVisibility = !this.socialExpEnabled;
+				this.lenderCountVisibility = true;
```

Comment:
This should be updated on line https://github.com/kiva/ui/pull/4458/files#diff-09ca3cc753df5d76e7f1404c47aa4ee2f07ce9ced29b352f4739cce2c1db3168R619 as well

Reply:
Good call. I had to redo the merge and missed this on the second go. That link doesn't seem to go to a line, but I'm assuming you were referring to line 619 (see below), and we just need to remove the `!this.socialExpEnabled`.

Old
```
if (this.status === 'fundraising' && this.numLenders > 0 && !this.socialExpEnabled) {
```

New
```
if (this.status === 'fundraising' && this.numLenders > 0) {
```

Reply:
Change pushed, and I'll go ahead and merge. Thanks!